linera-explorer/html/block.html

```diff
@@ -0,0 +1,115 @@
+        <i class="bi bi-braces"></i>
+      </button>
+      <div :id="block.hash+'-modal'" class="modal fade">
+        <div class="modal-dialog modal-xl">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title">{{ title }}</h5>
+              <button type="button" class="btn-close" data-bs-dismiss="modal"></button>
+            </div>
+            <div class="modal-body">
+              <div :id="block.hash+'-json'" style="overflow-x: auto"></div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </h5>
+    <ul class="list-group">
+      <li class="list-group-item d-flex justify-content-between">
+        <span><strong>Hash</strong></span>
+        {{ block.hash }}
+      </li>
+      <li class="list-group-item d-flex justify-content-between">
+        <span><strong>Epoch</strong></span>
+        <span>{{ block.value.executedBlock.block.epoch }}</span>
+      </li>
+      <li class="list-group-item d-flex justify-content-between">
+        <span><strong>Height</strong></span>
+        <span>{{ block.value.executedBlock.block.height }}</span>
+      </li>
+      <li class="list-group-item d-flex justify-content-between">
+        <span><strong>Timestamp</strong></span>
+        <span>{{ (new Date(block.value.executedBlock.block.timestamp/1000)).toLocaleString() }}</span>
+      </li>
+      <li class="list-group-item d-flex justify-content-between">
+        <span><strong>Signer</strong></span>
+        <span>{{ block.value.executedBlock.block.authenticatedSigner }}</span>
+      </li>
+      <li class="list-group-item d-flex justify-content-between">
+        <span><strong>Previous Block</strong></span>
+        <a v-if="block.value.executedBlock.block.previousBlockHash!==undefined" @click="route('block', [['block', block.value.executedBlock.block.previousBlockHash]])" class="btn btn-link">{{ block.value.executedBlock.block.previousBlockHash }}</a>
+        <span v-else>--</span>
+      </li>
+      <li class="list-group-item d-flex justify-content-between">
+        <span><strong>State Hash</strong></span>
+        <span>{{ block.value.executedBlock.stateHash }}</span>
+      </li>
+      <li class="list-group-item d-flex justify-content-between">
+        <span><strong>Status</strong></span>
+        <span>{{ block.value.status }}</span>
+      </li>
+      <li v-if="block.value.executedBlock.block.incomingMessages.length!==0" class="list-group-item d-flex justify-content-between" data-bs-toggle="collapse" :data-bs-target="'#in-messages-collapse-'+block.hash">
+        <span><strong>Incoming Messages</strong> ({{ block.value.executedBlock.block.incomingMessages.length }})</span>
+        <i class="bi bi-caret-down-fill"></i>
+      </li>
+      <li v-else class="list-group-item d-flex justify-content-between">
+        <span><strong>Incoming Messages</strong> (0)</span>
```

Comment:
(This if-else is because it would otherwise be displayed as `null`?)

Reply:
it's to avoid having a dropdown with nothing in it